### PR TITLE
Add vanilla-lazyload@2.1.2 w/ git auto-update

### PR DESCRIPTION
**Pull request by Piicksarn**
cc @cdnjs/intern, @cdnjs/intern2

Dear all, 
I use git auto-update due to the versions hosted on GitHub be much more complete than which hosted on NPM.
And pull request for issue: #9490.
Please help me for having a check, thank you!

-------

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/verlok/lazyload
 * Official website (optional, not the repository): http://verlok.github.io/lazyload/
 * NPM package url (optional): https://www.npmjs.com/package/vanilla-lazyload
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 596
   - Count of watchers: 26
   - Count of forks: 59
 * NPM download stats (optional):
   - Downloads in the last day: 4
   - Downloads in the last week: 11
   - Downloads in the last month: 525

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/vanilla-lazyload/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "description": "LazyLoad is a fast, lightweight and flexible script for loading images only when they're about to enter the viewport of a scrollable area, with an excellent support to the progressive JPEG image format. Typescript module definition is available.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/verlok/lazyload.git"
+  },
+  "license": "MIT",
+  "authors": [
+    "Andrea",
+    "verlok",
+    " Verlicchi"
```
Comment (maruilian11) [inline ajax/libs/vanilla-lazyload/package.json]: why leave a space?

File: ajax/libs/vanilla-lazyload/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  ],
+  "keywords": [
+    "lazy load",
```
Comment (maruilian11) [inline ajax/libs/vanilla-lazyload/package.json]: use a word

**Comment by PeterDaveHello** [comment]
@verlok once we finished the review and fix the parts, will be within few days.

File: ajax/libs/vanilla-lazyload/package.json (hunk 2 above)
Comment (Piicksarn) [inline ajax/libs/vanilla-lazyload/package.json]: this doesn't matter for using words.

**Comment by Piicksarn** [comment]
@cdnjs/intern @cdnjs/intern2,
I have it modified, and need ur help for reviewing.
Thank you all! :sweat_drops:

File: ajax/libs/vanilla-lazyload/package.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "source": "git",
+    "target": "git://github.com/verlok/lazyload.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "+(src|dist)/lazyload*"
+        ]
+      }
+    ]
```
Comment (pvnr0082t) [inline ajax/libs/vanilla-lazyload/package.json]: @Piicksarn Separate `lazyload*` files doesn't make sense. I suggest you can use this config:
```json
"fileMap": [
  {
    "basePath": "dist",
    "files": [
      "**/*"
    ]
  },
  {
    "basePath": "src",
    "files": [
      "lazyload*"
    ]
  }
]
```
Thanks.

**Comment by Piicksarn** [comment]
@cdnjs/intern, @cdnjs/intern2
I have auto-update config modified.
I need your reviewing. 😥 
Thank you all! 👏

**Comment by pvnr0082t** [comment]
LGTM

File: ajax/libs/vanilla-lazyload/package.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    "target": "git://github.com/verlok/lazyload.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
```
Comment (maruilian11) [inline ajax/libs/vanilla-lazyload/package.json]: `src` can't be the basePath, if you test this config, you won't get `src/lazyload.js`, it's always good to test your config after you change it.

Comment (pvnr0082t) [inline ajax/libs/vanilla-lazyload/package.json]: @maruilian11 I suggested the config due to the result should look like the following in this case:
```
$ tree 2.1.2/
2.1.2/
├── lazyload.js
├── lazyload.min.js
└── lazyload.min.js.map
```

Comment (PeterDaveHello) [inline ajax/libs/vanilla-lazyload/package.json]: This one is correct
```
$ tree 2.1.2/
2.1.2/
├── lazyload.js
├── lazyload.min.js
└── lazyload.min.js.map
```
That's why we need multi filemap ...

Comment (maruilian11) [inline ajax/libs/vanilla-lazyload/package.json]: thanks @pvnr0082t, I understand.

Comment (PeterDaveHello) [inline ajax/libs/vanilla-lazyload/package.json]: Then why did you just approve this PR without further updates? @maruilian11 ? Think clearly before your movement please.

**Comment by maruilian11** [comment]
@Piicksarn please use current auto-update config to catch the files again, you will get different file structures from now.

**Comment by Piicksarn** [comment]
@cdnjs/intern, @cdnjs/intern2
I have auto-update config modified, and need your reviewing.
Thank you all! 👏

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**